vuepress/docs/.vuepress/next.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
                 title: 'Introduction',
                 children: [
                     {
-                        title: 'Overview',
+                        title: 'Entando Platform',
```

Comment:
I think we should keep it simpler (switching #3 and #4).
- Overview
- Getting Started
- Solution Template
- Development Process
- Entando Architecture

It looks like we need to add Solution Templates to 6.3.2?

Reply:
@jyunmitch I don't think these changes made it in. I'm adding a PR with that and a couple other suggestions.

Reply:
https://github.com/entando/entando-docs/pull/417/files